filebeat/module/haproxy/_meta/config.yml

```diff
@@ -0,0 +1,8 @@
+- module: haproxy
+  # All logs
+  {fileset}:
```

Comment:
This should be replaced by the name of the fileset, I guess `http` in this case.

Reply:
:+1: